admin_manual/configuration_server/occ_command.rst

```diff
@@ -72,18 +94,197 @@ this example for the ``maintenance:mode`` command::
    --no-ansi             Disable ANSI output.
    --no-interaction (-n) Do not ask any interactive question.
    
+Apps Commands
+-------------
+
+The ``app`` commands list, enable, and disable apps. This lists all of your 
+installed apps, and shows whether they are enabled or disabled::
+
+ $ sudo -u www-data php occ app:list
+ 
+Enable an app::
+
+ $ sudo -u www-data php occ app:enable external
+   external enabled
+   
+``app:check-code`` checks if the app uses ownCloud's public API (``OCP``) or 
+private API (``OC_``), and then enables the app. If the app uses the private 
```

Comment:
@nickvergessen I just described the behavior. Something's weird with it anyway, because pretty much all the official ownCloud apps fail the check.